docs/address_lookup/a_brief_history_of_postcodes.md

```diff
@@ -0,0 +1,13 @@
+# A brief history of postcodes
+
+Postcodes in the UK date back to 1857 and originated in London.  A project team, led by team Sir Rowland Hill, set out to solve a problem that was identified the previous year where 100 million items of post were sent to a population of around 3 million over an area with a radius of ~12 miles centred around the St Paul's Cathedral post office.  The postal service was struggling!
+
+![Sir Rowland Hill](images/IMG_0205.jpg)
+
+<p><center><strong>Sir Rowland Hill</strong></center></p>
+
+To solve this problem the team originally divided London into 10 districts - 1 district per standard compass point (N, NE, E, SE, S, SW, W, NW), and 2 districts for the centre of London - East Central (EC) and West Central (WC).  However, in 1867 the NE district was controversially removed with all residents being absorbed into the E district, and in 1868 the South district was also removed with their residents being merged either into the South East (SE) or South West (SW) district. This left London with the the following districts: N, E, SE, SW, W, NW, EC and WC.
+
+The general approach of splitting into districts was subsequently adopted by Liverpool in 1864/5, then Manchester and Salford in 1867/8. By 1934 Birmingham, Brighton and Hove, Bristol, Edinburgh, Glasgow, Leeds and Bradford, Newcastle upon Tyne and Sheffield had also adopted the same approach.
+
+By the early 1960's - as mail volumes soared - a re-think of the existing postcode system was required.  And so was automated mail sorting.  Following a trial in Norwich a new - mechanised - sorting process was introduced.  By 1970, the system had been rolled out across all major centres across the UK and the postcode system that we have today was finally in place.
```

Comment:
Sorry to be a party-pooper, please remove this file. In an emergency when I'm looking at fixing the address lookup feature, this will delay me.

Reply:
Removed

Reply:
Aww, I really liked it! It gave some humanity and context!